frontend/src/components/Header.tsx

```diff
@@ -0,0 +1,12 @@
+import "./Header.scss"
+
+interface headerProps{
+    name: string
+}
+
+export default function Header(props: headerProps){
```

Comment:
Since this Header has only one tag and not much CSS, maybe you can get rid of it or combine it with some kind of page layout. Not wrong, but you#ll get a ton of components like that.